docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md

```diff
@@ -56,14 +53,17 @@ Example of using ``example_input``:
 * ``list`` or ``tuple`` with tensors (``openvino.runtime.Tensor`` / ``torch.Tensor`` / ``np.ndarray``)
 * ``dictionary`` where key is the input name, value is the tensor (``openvino.runtime.Tensor`` / ``torch.Tensor`` / ``np.ndarray``)
 
+Sometimes ``convert_model`` will produce inputs of the model with dynamic rank or dynamic type, such model may not be supported by hardware plugin,
+to solve this, please use ``input`` argument of ``convert_model``, please refer to `Convert Models Represented as Python Objects <openvino_docs_MO_DG_Python_API>` for more information. 
```

Comment:
```suggestion
Sometimes ``convert_model`` will produce inputs of the model with dynamic rank or dynamic type. 
Such model may not be supported by the hardware chosen for inference. To avoid this issue,
use the ``input`` argument of ``convert_model``. For more information, refer to `Convert Models Represented as Python Objects <openvino_docs_MO_DG_Python_API>`. 
```